cedar-policy-generators/src/expr.rs

```diff
@@ -566,7 +566,7 @@ impl<'a> ExprGenerator<'a> {
                                             // This does not use an explicit namespace because entity types
                                             // implicitly use the schema namespace if an explicit one is not
                                             // provided.
-                                            name: entity_name.clone().into(),
+                                            name: ast::Name::from(entity_name.clone()).into(),
```

Comment:
`ast::Name::from(entity_name.clone()).into()` is used a couple places in this PR. Without having thought about it too much -- is there a way to do this without cloning? What is it converting _into_?

Reply:
Cloning a `Name` is actually very cheap.